[debug] [2023-12-17T10:08:00.441Z] ----------------------------------------------------------------------
[debug] [2023-12-17T10:08:00.451Z] Command:       /usr/local/bin/node /usr/local/bin/firebase deploy
[debug] [2023-12-17T10:08:00.451Z] CLI Version:   12.6.2
[debug] [2023-12-17T10:08:00.451Z] Platform:      darwin
[debug] [2023-12-17T10:08:00.451Z] Node Version:  v20.3.1
[debug] [2023-12-17T10:08:00.451Z] Time:          Sun Dec 17 2023 16:08:00 GMT+0600 (GMT+06:00)
[debug] [2023-12-17T10:08:00.451Z] ----------------------------------------------------------------------
[debug] 
[debug] [2023-12-17T10:08:00.511Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2023-12-17T10:08:00.511Z] > authorizing via signed-in user (asibhasanriyad@gmail.com)
[debug] [2023-12-17T10:08:00.511Z] [iam] checking project connectopia-9e807 for permissions ["firebase.projects.get","firebasehosting.sites.update"]
[debug] [2023-12-17T10:08:00.513Z] >>> [apiv2][query] POST https://cloudresourcemanager.googleapis.com/v1/projects/connectopia-9e807:testIamPermissions [none]
[debug] [2023-12-17T10:08:00.513Z] >>> [apiv2][(partial)header] POST https://cloudresourcemanager.googleapis.com/v1/projects/connectopia-9e807:testIamPermissions x-goog-quota-user=projects/connectopia-9e807
[debug] [2023-12-17T10:08:00.513Z] >>> [apiv2][body] POST https://cloudresourcemanager.googleapis.com/v1/projects/connectopia-9e807:testIamPermissions {"permissions":["firebase.projects.get","firebasehosting.sites.update"]}
[debug] [2023-12-17T10:08:01.889Z] <<< [apiv2][status] POST https://cloudresourcemanager.googleapis.com/v1/projects/connectopia-9e807:testIamPermissions 200
[debug] [2023-12-17T10:08:01.890Z] <<< [apiv2][body] POST https://cloudresourcemanager.googleapis.com/v1/projects/connectopia-9e807:testIamPermissions {}
[debug] [2023-12-17T10:08:01.890Z] [iam] error while checking permissions, command may fail: FirebaseError: Authorization failed. This account is missing the following required permissions on project connectopia-9e807:

  firebase.projects.get
  firebasehosting.sites.update
[debug] [2023-12-17T10:08:01.890Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects/connectopia-9e807 [none]
[debug] [2023-12-17T10:08:02.427Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects/connectopia-9e807 403
[debug] [2023-12-17T10:08:02.427Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects/connectopia-9e807 {"error":{"code":403,"message":"The caller does not have permission","status":"PERMISSION_DENIED"}}
[debug] [2023-12-17T10:08:02.429Z] HTTP Error: 403, The caller does not have permission
[debug] [2023-12-17T10:08:02.603Z] FirebaseError: HTTP Error: 403, The caller does not have permission
    at responseToError (/usr/local/lib/node_modules/firebase-tools/lib/responseToError.js:49:12)
    at RetryOperation._fn (/usr/local/lib/node_modules/firebase-tools/lib/apiv2.js:285:77)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[error] 
[error] Error: Failed to get Firebase project connectopia-9e807. Please make sure the project exists and your account has permission to access it.
[debug] [2023-12-17T10:08:28.359Z] ----------------------------------------------------------------------
[debug] [2023-12-17T10:08:28.362Z] Command:       /usr/local/bin/node /usr/local/bin/firebase deploy
[debug] [2023-12-17T10:08:28.362Z] CLI Version:   12.6.2
[debug] [2023-12-17T10:08:28.362Z] Platform:      darwin
[debug] [2023-12-17T10:08:28.362Z] Node Version:  v20.3.1
[debug] [2023-12-17T10:08:28.362Z] Time:          Sun Dec 17 2023 16:08:28 GMT+0600 (GMT+06:00)
[debug] [2023-12-17T10:08:28.362Z] ----------------------------------------------------------------------
[debug] 
[debug] [2023-12-17T10:08:28.420Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2023-12-17T10:08:28.421Z] > authorizing via signed-in user (asibhasanriyad@gmail.com)
[debug] [2023-12-17T10:08:28.421Z] [iam] checking project connectopia-9e807 for permissions ["firebase.projects.get","firebasehosting.sites.update"]
[debug] [2023-12-17T10:08:28.422Z] >>> [apiv2][query] POST https://cloudresourcemanager.googleapis.com/v1/projects/connectopia-9e807:testIamPermissions [none]
[debug] [2023-12-17T10:08:28.422Z] >>> [apiv2][(partial)header] POST https://cloudresourcemanager.googleapis.com/v1/projects/connectopia-9e807:testIamPermissions x-goog-quota-user=projects/connectopia-9e807
[debug] [2023-12-17T10:08:28.423Z] >>> [apiv2][body] POST https://cloudresourcemanager.googleapis.com/v1/projects/connectopia-9e807:testIamPermissions {"permissions":["firebase.projects.get","firebasehosting.sites.update"]}
[debug] [2023-12-17T10:08:29.957Z] <<< [apiv2][status] POST https://cloudresourcemanager.googleapis.com/v1/projects/connectopia-9e807:testIamPermissions 200
[debug] [2023-12-17T10:08:29.958Z] <<< [apiv2][body] POST https://cloudresourcemanager.googleapis.com/v1/projects/connectopia-9e807:testIamPermissions {}
[debug] [2023-12-17T10:08:29.958Z] [iam] error while checking permissions, command may fail: FirebaseError: Authorization failed. This account is missing the following required permissions on project connectopia-9e807:

  firebase.projects.get
  firebasehosting.sites.update
[debug] [2023-12-17T10:08:29.959Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects/connectopia-9e807 [none]
[debug] [2023-12-17T10:08:30.524Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects/connectopia-9e807 403
[debug] [2023-12-17T10:08:30.524Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects/connectopia-9e807 {"error":{"code":403,"message":"The caller does not have permission","status":"PERMISSION_DENIED"}}
[debug] [2023-12-17T10:08:30.526Z] HTTP Error: 403, The caller does not have permission
[debug] [2023-12-17T10:08:30.741Z] FirebaseError: HTTP Error: 403, The caller does not have permission
    at responseToError (/usr/local/lib/node_modules/firebase-tools/lib/responseToError.js:49:12)
    at RetryOperation._fn (/usr/local/lib/node_modules/firebase-tools/lib/apiv2.js:285:77)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[error] 
[error] Error: Failed to get Firebase project connectopia-9e807. Please make sure the project exists and your account has permission to access it.
